compiler/x/codegen/BinaryEvaluator.cpp

```diff
@@ -1252,8 +1252,7 @@ TR::Register *OMR::X86::TreeEvaluator::caddEvaluator(TR::Node *node, TR::CodeGen
       }
 
    TR::Register * testRegister = secondChild->getRegister();
-   if (secondChild->getOpCodeValue() == TR::cconst &&
```

Comment:
This whole if statement can be removed. The condition is always false as the TR::cconst no longer exists

Reply:
Yeah I think this should be sconst with appropriate updates - I don't think the whole case can be removed

Reply:
Fixed in c215804.